Update links.yml

ratkinson0405 opened the pull request:
Update the links to the 'Using Supercomputers - Parts 1 & 2'.

ratkinson0405 committed:
Update links.yml

marcodelapierre reviewed (changes requested):
Hi Rob, just a couple of comments:

1- add "git: https://github.com/PawseySC/Introductory-Supercomputing" to both trainings, as they both use these github materials

2- right now the youtube "vid" links point to the same video, this needs fixing. I just checked on youtube and noticed that right now the order of the videos in that playlist is wrong and needs fixing, too, so these links will need to change again. Maybe fix the links with the current youtube config, and I will keep you in the loop on the youtube fix.

ratkinson0405 committed:
Update links.yml

ratkinson0405 commented:
Hi Marco, I've added the github materials links. I have also updated the vid links, but they appeared to link to the same videos. I may need to clarify the vid links with you.

marcodelapierre committed:
Updated youtube links for intro to SC 2x

marcodelapierre commented:
thanks Rob, I have just done a further edit to the youtube links, so that watching starts from the beginning rather than from a custom time
